content/ri/using-redisinsight/api-get.md

```diff
@@ -0,0 +1,69 @@
+---
+Title: Adding Databases via GET URL
+date: 2021-01-11 00:00:00 +0000
+weight: 16
+categories: ["RI"]
+path: api/
+nextStep:
+    Title: Memory Analysis
+    href: /docs/features/memory-analysis/
+---
+
+If you want to automate adding a Redis database without filling the [database form]({{< relref "/ri/using-redisinsight/add-instance.md" >}}), you might want to add database via GET URL.
```

Comment:
When you say "via GET URL", where are you running this `GET` command?

Reply:
Not sure whether you mean the [Redis GET command](https://redis.io/commands/get). This has nothing to with that.

Any URL you hit by entering the URL in your browser's address bar is an HTTP GET request. So "via GET URL" basically translates to "by visiting this URL in your browser".

Reply:
So if the users are just typing this into their browser, how do they automate it? Is the URL something they can include in a program or script?

Reply:
This is not meant for automation. It's useful in a classroom setting where the instructor can just share a GET URL with the class and the students can just follow the link to add the Redis database.
For proper automation using scripts, the [documented API](https://docs.redislabs.com/latest/ri/using-redisinsight/api/) should be used.

Reply:
Are there any other use cases for this method besides a classroom setting?

Reply:
I suppose it's any case where you want to add the Redis database easily, for example, a devops team.